packages/fast-components-react-base/src/image/image.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, { HandledProps } from "../foundation";
+import { IImageHandledProps, IImageMangedClasses, IImageUnhandledProps } from "./image.props";
+import { IImageClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts";
+
+/**
+ * TODO #239 #240: Hook viewports values into Styles
+ */
+export interface IBreakpoints {
+    vp1: number;
+    vp2: number;
+    vp3: number;
+    vp4: number;
+    vp5: number;
+    vp6: number;
+}
+
+const breakpoints: IBreakpoints = {
+    vp1: 0,
+    vp2: 540,
+    vp3: 768,
+    vp4: 1084,
+    vp5: 1400,
+    vp6: 1779
+};
+
+/* tslint:disable-next-line */
+class Image extends Foundation<IImageHandledProps & IManagedClasses<IImageClassNameContract>, React.HTMLAttributes<HTMLImageElement | HTMLPictureElement>, {}> {
+    protected handledProps: HandledProps<IImageHandledProps & IManagedClasses<IImageClassNameContract>> = {
+        managedClasses: void 0,
+        alt: void(0),
+        itemScope: void(0),
+        round: void(0),
+        src: void(0),
```

Comment:
Are we still passing srcSet and sizes? Should be enumerated here. We likely only want to add these to image and not <picture>. My assumption would be that srcset would simply be applied in picture by passing that value in vp1, vp2, etc...as that is for the srcSet attribute.

Reply:
That's correct, Adding those now. Correct on passing srcset string to vp1, vp2, etc.